react/features/virtual-background/components/VirtualBackgroundDialog.js

```diff
@@ -0,0 +1,128 @@
+// @flow
+/* eslint-disable react/jsx-no-bind, no-return-assign */
+import React, { useState } from 'react';
+
+import { Dialog } from '../../base/dialog';
+import { translate } from '../../base/i18n';
+import { Icon, IconBlurBackground } from '../../base/icons';
+import { connect } from '../../base/redux';
+import { Tooltip } from '../../base/tooltip';
+import { toggleBackgroundEffect } from '../../blur/actions';
+import { setVirtualBackground } from '../actions';
+
+const images = [
+    {
+        tooltip: 'Image 1',
+        name: 'background-1.jpg',
+        id: 1,
+        src: 'images/virtual-background/background-1.jpg'
+    },
+    {
+        tooltip: 'Image 2',
+        name: 'background-2.jpg',
+        id: 2,
+        src: 'images/virtual-background/background-2.jpg'
+    },
+    {
+        tooltip: 'Image 3',
+        name: 'background-3.jpg',
+        id: 3,
+        src: 'images/virtual-background/background-3.jpg'
+    },
+    {
+        tooltip: 'Image 4',
+        name: 'background-4.jpg',
+        id: 4,
+        src: 'images/virtual-background/background-4.jpg'
+    }
+];
+type Props = {
+
+    /**
+     * The redux {@code dispatch} function.
+     */
+    dispatch: Function,
+
+    /**
+     * The JitsiLocalTrack to display.
+     */
+    videoTrack: ?Object
+};
+
+/**
+ * Renders virtual background dialog.
+ *
+ * @returns {ReactElement}
+ */
+function VirtualBackground({ dispatch }: Props) {
+    const selectedStyle = '2px solid #a4b8d1';
```

Comment:
Define this in CSS since that's where most of the styling for this is. You can add an extra class name when selected.